dynaconf/validator.py

```diff
@@ -229,6 +232,24 @@ def __init__(self, validator_a, validator_b, *args, **kwargs):
         """Takes 2 validators and combines the validation"""
         self.validators = (validator_a, validator_b)
         super().__init__(*args, **kwargs)
+        self.names = self.names or tuple(
+            validator.names for validator in self.validators
+        )
+        self.must_exist = self.must_exist or tuple(
+            validator.must_exist for validator in self.validators
+        )
+        self.when = self.when or tuple(
+            validator.when for validator in self.validators
+        )
+        self.condition = self.condition or tuple(
+            validator.condition for validator in self.validators
+        )
+        self.operations = self.operations or tuple(
+            validator.operations for validator in self.validators
+        )
+        self.envs = self.envs or tuple(
+            validator.envs for validator in self.validators
+        )
```

Comment:
what if on the top of the file we define a constant

```py
EQUALITY_ATTRS = ("names", "must_exist", "when", "condition", "operations", "envs")
```

then we can use that on line [133](https://github.com/rochacbruno/dynaconf/pull/413/files#diff-2f5568fbbe07143ea6573f6f64c74bbdR133)

and then here as most of this code is pretty similar we also reuse that attrs?

```suggestion
        for attr in EQUALITY_ATTRS:
            if getattr(self, attr, None) is None:
                setattr(
                    self, 
                    attr,
                    tuple(getattr(validator, attr) for validator in self.validators)
                ) 
```

It is not very readable, but if we add more attrs in the `EQUALITY_ATTRS` list in the future it will be dynamically covered.

Reply:
Added in 72a2b76c515271d6eaf2f52d38b9f692260d473f . Should I squash?